perses/app/relative_point_mutation_setup.py

```diff
@@ -25,7 +25,6 @@
 
 # Set up logger
 _logger = logging.getLogger()
-_logger.setLevel(logging.DEBUG)
```

Comment:
I would leave this as 

`_logger.setLevel(logging.INFO)`

Reply:
i agree, will change back